src/setupTests.ts

```diff
@@ -9,6 +9,13 @@ import { enableFetchMocks } from "jest-fetch-mock";
 import { createIdentifier } from "./fhir/resource_helpers";
 import { createPractitioner } from "./fhir/testUtilities";
 
+jest.resetAllMocks();
+export const mockedNavigate = jest.fn();
+jest.mock("react-router-dom", () => ({
```

Comment:
Maybe add a comment here to say that we're mocking navigate to be able to determine redirects in component tests?